models/eprop_iaf.cpp

```diff
@@ -214,6 +229,18 @@ eprop_iaf::Parameters_::set( const DictionaryDatum& d, Node* node )
     throw BadProperty( "Cutoff of integration of eprop trace between spikes eprop_isi_trace_cutoff ≥ 0 required." );
   }
 
+  if ( delay_rec_out_ < 1 )
+  {
+    throw BadProperty( "Connection delay from recurrent to output neuron ≥ 1 required." );
+  }
+
+  if ( delay_out_rec_ < 1 )
+  {
+    throw BadProperty( "Broadcast delay of learning signals ≥ 1 required." );
+  }
```

Comment:
We did not establish the term broadcast in our documentation, so maybe clearer to refer to the location (readout to recurrent) of the synapse in the error message.
```suggestion
  {
    throw BadProperty( "Connection delay from readout to recurrent neuron ≥ 1 required." );
  }
```